### PR TITLE
Update ANNOUNCE for v1.8

### PR DESCRIPTION
**Pull request by langner**

### CODE REVIEW

File: ANNOUNCE [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+On behalf of the cclib development team, we are pleased to announce the release of cclib 1.8, which is now available for download from https://cclib.github.io. This is a major update that includes Julia bindings, two new parsers (NBO and GAMESS dat files), several new attributes and multiple other improvements and fixes.
 
-cclib is an open source library, written in Python, for parsing and interpreting the results of computational chemistry packages. It currently parses output files from 15 different programs: ADF, DALTON, Firefly, GAMESS (US), GAMESS-UK, Gaussian, Jaguar, Molpro, MOLCAS, MOPAC, NWChem, ORCA, Psi, QChem and Turbomole.
+cclib is an open source library, written in Python, for parsing and interpreting the results of computational chemistry packages. It currently parses output files from 15 different programs: ADF, DALTON, Firefly, GAMESS (US), GAMESS-UK, Gaussian, Jaguar, Molpro, MOLCAS, MOPAC, NBO, NWChem, ORCA, Psi, QChem and Turbomole.
```
Comment (shivupa) [inline ANNOUNCE:3]: 16 different programs now

**Review by shivupa — APPROVED** [review]
Minor edit

**Review by berquist — APPROVED** [review]

File: ANNOUNCE [hunk 2]
```diff
@@ -9,33 +9,27 @@ Among other data, cclib extracts:
...
-
 If you need help, find a bug, want new features or have any questions, please send an email to our mailing list:
     https://lists.sourceforge.net/lists/listinfo/cclib-users
-
 If your published work uses cclib, please support its development by citing the following article:
     N. M. O'Boyle, A. L. Tenderholt, K. M. Langner, cclib: a library for package-independent computational chemistry algorithms, J. Comp. Chem. 29 (5), 839-845 (2008)
 You can also specifically reference this version of cclib as:
-    Eric Berquist, Geoffrey Hutchison, Karol M. Langner, Noel M. O'Boyle, Adam L. Tenderholt, Shiv Upadhyay. Release of cclib version 1.7. 2020. https://dx.doi.org/10.5281/zenodo.4420433
-
+    Eric Berquist, Geoffrey Hutchison, Karol M. Langner, Noel M. O'Boyle, Adam L. Tenderholt, Shiv Upadhyay. Release of cclib version 1.7. 2020. https://doi.org/10.5281/zenodo.4420433
```
Comment (berquist) [inline ANNOUNCE:24]: It looks like we have a chicken-and-egg problem with minting a new (Zenodo) DOI and updating this file.  We will have to update it again after tagging and uploading to Zenodo.

Comment (langner) [reply ANNOUNCE:24]: Indeed, always frustrated me before. Thanks for merging.

**Pull request merged**